 Always force absolute path for LD_PRELOAD.
- Make getopt test work with busybox implementation.
---
diff --git scripts/fakeroot.in scripts/fakeroot.in
index 0d6f435..148b9c7 100755
--- scripts/fakeroot.in
+++ scripts/fakeroot.in
@@ -33,7 +33,7 @@ fatal ()
 FAKEROOT_PREFIX=@prefix@
 FAKEROOT_BINDIR=@bindir@
 
-USEABSLIBPATH=@LDPRELOADABS@
+USEABSLIBPATH=1
 FAKEROOT_LIB=lib@fakeroot_transformed@@DLSUFFIX@
 PATHS=@libdir@:${FAKEROOT_PREFIX}/lib64/libfakeroot:${FAKEROOT_PREFIX}/lib32/libfakeroot
 FAKED=${FAKEROOT_BINDIR}/@faked_transformed@
